src/QirRuntime/README.md

```diff
@@ -11,24 +12,17 @@ This folder contains QIR runtime project, which includes implementation of the
 
 ## Build
 
+### Prerequisites
 The QirRuntime project is using CMake (3.17) + Ninja(1.10.0) + Clang++(11.0.0). Other versions of the tools might work
- but haven't been tested. Only x64 architecture is supported.
+ but haven't been tested. Only x64 architecture is supported.  
+For runing the PowerShell scripts below use the powershell core (`pwsh`),
+e.g. [this one](https://github.com/PowerShell/PowerShell/releases/download/v7.1.2/PowerShell-7.1.2-win-x64.msi), not the inbox PowerShell.
 
-You can use CMake directly. For example, to produce a release build:
+To install prerequisite tools for building the QIR runtime, you can set the `ENABLE_QIRRUNTIME` environment variable to the string `"true"` 
+and run [`prerequisites.ps1`](prerequisites.ps1), or manually install pre-reqs with the steps listed below
+(Windows steps rely on [Chocolatey](https://chocolatey.org/) for installation).
```

Comment:
```suggestion
and run [`prerequisites.ps1`](prerequisites.ps1), or manually install pre-reqs with the steps listed below.
Note that on Windows, this script relies on the [Chocolatey package manager](https://chocolatey.org/), while on macOS, `prerequisites.ps1` relies on the `brew` package manager.
```

A link to homebrew may also be helpful here.

Reply:
> A link to homebrew may also be helpful here.

I assume it is https://brew.sh